server/util/file.py

```diff
@@ -44,12 +45,22 @@ def s3():  # pylint: disable=invalid-name
     )
 
 
+class Progress:
+    def __init__(self):
+        self.bytes_processed = 0
+
+    def __call__(self, new_bytes_processed):
+        self.bytes_processed += new_bytes_processed
+        sys.stdout.write("\r%d bytes processed" % self.bytes_processed)
+        sys.stdout.flush()
+
+
 def store_file(file: IO[bytes], storage_path: str) -> str:
     assert not os.path.isabs(storage_path)
     full_path = os.path.join(config.FILE_UPLOAD_STORAGE_PATH, storage_path)
     if config.FILE_UPLOAD_STORAGE_PATH.startswith("s3://"):
         bucket_name = urlparse(config.FILE_UPLOAD_STORAGE_PATH).netloc
-        s3().upload_fileobj(file, bucket_name, storage_path)
+        s3().upload_fileobj(file, bucket_name, storage_path, Callback=Progress())
```

Comment:
What the progress indicator looks like, for reference!

![bytes-processed](https://user-images.githubusercontent.com/12616928/203456082-8a382b43-ba1d-45a4-8d63-b70159e06100.gif)